src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -73,17 +74,16 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
 
     forms.forEach(form => DynamicFormService.updateValidity(form, errors));
 
-    errors.length
-      ? this.errorService.setErrors(errors)
-      : this.errorService.clearErrors();
+    errors.length ? this.errorService.setErrors(errors) : this.errorService.clearErrors();
 
     return forms.some(form => form.invalid);
   }
 
   handleFormState() {
     const form = this.summary.sections
       .map(section => section.form)
-      .concat(this.summary.dimensions.form);
+      .concat(this.summary.dimensions.form)
+      .concat(this.summary.weights.form);
```

Comment:
`.concat()` accepts an input parameter of `...items: T[]`, so you can pass multiple values separated by a comma.
```suggestion
      .concat(this.summary.dimensions.form, this.summary.weights.form);
```